week1/1.5/Notes/Notes/Controllers/View Controllers/NoteDetailViewController.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  NoteDetailViewController.swift
+//  Notes
+//
+//  Created by Eric Lanza on 7/6/18.
+//  Copyright © 2018 ETLanza. All rights reserved.
+//
+
+import UIKit
+
+class NoteDetailViewController: UIViewController {
+ 
+    //MARK: - IBOutlet
+    @IBOutlet weak var bodyTextView: UITextView!
+    
+    //MARK: - IBAction
+    @IBAction func saveButtonPressed(_ sender: UIBarButtonItem) {
+        guard let text = bodyTextView.text, text != "" else { return }
```

Comment:
Another user unfriendly failure. A button shouldn't be active and not responsive. You could disable the button fi there is no text or you could open a quick popup saying "There's not text here" or something.